libs/components/indicators/src/lib/modules/help-inline/help-inline.component.ts

```diff
@@ -1,17 +1,52 @@
-import { Component, EventEmitter, Output } from '@angular/core';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
 
 @Component({
   selector: 'sky-help-inline',
   templateUrl: './help-inline.component.html',
   styleUrls: ['./help-inline.component.scss'],
 })
 export class SkyHelpInlineComponent {
+  /**
+   * The ID to identify the element that the help inline button exposes.
```

Comment:
I'd change this to "The ID of the element that the help inline button controls."

Reply:
Done